edi-tools/modules/codegen/maincode_gen.bal

```diff
@@ -29,6 +29,11 @@ public isolated function getEDINames() returns string[] {
     return ${libdata.ediNames.toString()};
 }
 
+# Convert EDI string to Ballerina record.
+# 
+# + ediText - EDI string to be converted
+# + ediName - EDI type name
+# + return - Ballerina record
```

Comment:
I see the return type as anydata|error so this doc comment is confusing

Reply:
Changed with 1330f90e144a8514f5a75f7fd1d041a44035e7ba.